terraform/eks/e2e/main.tf

```diff
@@ -52,6 +52,15 @@ resource "aws_eks_node_group" "this" {
   ]
 }
 
+resource "aws_security_group_rule" "nodeport_inbound" {
```

Comment:
Why is this required ?

Reply:
We need this to be able to access the service that the Tomcat application uses in order to send load to it.

ref: https://github.com/aws/amazon-cloudwatch-agent-test/blob/8cb81ce05f18b584b05c7317295b6379bcd780f6/test/e2e/jmx/resources/sample_apps/tomcat.yaml#L18